### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by StellaAthena**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,9 +242,11 @@ Coming soon! Check out the `distill-gpt-neox` branch to try distilling a model.
 
 ## Monitoring
 
-### WandB
+In addition to storing logs locally, we provide built-in support for two popular experiment monitoring Weights & Biases 
 
-EleutherAI is currently using [Weights & Biases to record experiments](https://wandb.ai/eleutherai/neox). If you are logged into Weights & Biases on your machine - you can do this by executing `wandb login` - your runs will automatically be recorded. Additionally, set the config parameter `wandb_team` if you would like the run to be added to an organisation/team account.
+### Weights & Biases
```
Comment (EricHallahan) [inline README.md:247]: This will introduce issues with the Table of Contents. I have a fix in the works.

File: README.md [hunk 2]
```diff
@@ -242,9 +242,11 @@ Coming soon! Check out the `distill-gpt-neox` branch to try distilling a model.
 
 ## Monitoring
 
-### WandB
+In addition to storing logs locally, we provide built-in support for two popular experiment monitoring Weights & Biases
```
Comment (EricHallahan) [inline README.md:245]: Incomplete.

**Review by EricHallahan — CHANGES_REQUESTED** [review]
There are numerous issues that remain to be addressed. I'll submit an update shortly to resolve many of them.

**Comment by StellaAthena** [comment]
Whoops! I think I fixed the issues

**Pull request merged**